src/main/resources/RecruitmentApp/FieldsAdministration.xml

```diff
@@ -68,8 +68,8 @@
   $newDoc.save()
 #end
 {{/velocity}}
-{{velocity}}
 == Offices ==

```

Comment:
I placed Offices heading between the two code because we were getting an unwanted p tag otherwise.